calyx-frontend/src/ast.rs

```diff
@@ -14,8 +14,10 @@ pub struct NamespaceDef {
     pub components: Vec<ComponentDef>,
     /// Extern statements and any primitive declarations in them.
     pub externs: Vec<(Option<PosString>, Vec<Primitive>)>,
-    /// Optional opaque metadata
+    /// Optional legacy opaque metadata. Newer metadata is in [NamespaceDef::file_info_table]
```

Comment:
Please leave the comment as it was before. There might be other uses for the opaque metadata that are not covered by the new `SourceInfoTable`.